### ISSUE TITLE
Clearify UseNode@1 vs. NodeTool@0 usage

### ISSUE THREAD

**Issue by cremor** [MicrosoftDocs/azure-devops-yaml-schema#239]
It looks like `UseNode@1` is a newer version of `NodeTool@0`?

If this is the case then this page should be updated to mark this task as deprecated and link to `UseNode@1`.
Also, the pages https://learn.microsoft.com/en-us/azure/devops/pipelines/ecosystems/javascript and https://learn.microsoft.com/en-us/azure/devops/pipelines/process/tasks should then be updated to use `UseNode@1`.

If this is not the case then both pages should be updated to clearify the difference.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 07ed1dbe-4bd5-52a4-9373-32e1fbca211d
* Version Independent ID: 07ed1dbe-4bd5-52a4-9373-32e1fbca211d
* Content: [NodeTool@0 - Node.js tool installer v0 task](https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/node-tool-v0?view=azure-pipelines)
* Content Source: [task-reference/node-tool-v0.md](https://github.com/MicrosoftDocs/azure-devops-yaml-schema/blob/main/task-reference/node-tool-v0.md)
* Service: **azure-devops-pipelines**
* Sub-service: **azure-devops-pipelines-tasks**
* GitHub Login: @steved0x
* Microsoft Alias: **sdanie**

**Comment by WilliamAntonRohm** [comment]
Related to #238.

**Comment by steved0x** [comment]
Reactivating this one to track finding the difference between those two tasks and adding an update (or deprecation notice) to the article to provide that info.

**Comment by steved0x** [comment]
@cremor I just got some information - they are the same task, the newer one has a simplified set of task inputs. I believe the name change was due to changing the pattern of tool installers with a "Use" prefix.

I will add a link to the older task to let folks know there is a newer version and I will group them in the TOC and index so they are together. The older version isn't marked as deprecated, but if it does get deprecated, the docs will pick that change up.
